### PR TITLE
feat: support multi ca certs

### PR DESCRIPTION
**Pull request by joway**
Hi, I notice that mongo-go-driver cannot support multiple `CERTIFICATE` sections in .pem file, which supported by other mongo drivers like pymongo. 

Please let me know if you have any questions.

### CODE REVIEW

File: mongo/options/clientoptions.go [hunk 1]
```diff
@@ -735,59 +735,60 @@ func MergeClientOptions(opts ...*ClientOptions) *ClientOptions {
 	return c
 }
 
-// addCACertFromFile adds a root CA certificate to the configuration given a path
+// addCACertsFromFile adds a root CA certificate to the configuration given a path
```
Comment (craiggwilson) [inline mongo/options/clientoptions.go]: fix up comment: adds "root" CA certificates

**Comment by kchrusciel** [comment]
Any chance to have that?

**Comment by divjotarora** [comment]
Hi @kchrusciel,

There hasn't been progress on this PR because it's currently missing a test. I'm going to close this and open a new one with an additional test. Given this is a new feature and not a bugfix, we plan to release this in the next minor version release (1.4.0), which is due to be released in a couple of months.

**Comment by alexandrethiel** [comment]
Hi @divjotarora 
I can't find the new PR you mentionned and fix is not present in 1.4.0-beta1

**Comment by divjotarora** [comment]
Hi @alexandrethiel,

I've filed https://jira.mongodb.org/browse/GODRIVER-1580 to address this. 1.4.0-beta1 was a pre-release focused on supporting as many of the new server features for MongoDB 4.4 as we could. Now that the beta has happened, we have some more time to concentrate on tickets like this, so I'm hoping to pick it up very soon. Also, we looked further into the ticket and as discussed in the ticket description, we believe we can replace all of the existing certificate loading code with a single call to `AppendCertsFromPEM`, so we'll be investigating that approach.

-- Divjot

**Comment by divjotarora** [comment]
@alexandrethiel I've implemented the new solution in #389. Feel free to look over that one and ensure that it'll add support for your use case.

-- Divjot

**Pull request closed without merging**